src/main/java/com/nulabinc/zxcvbn/Zxcvbn.java

```diff
@@ -2,13 +2,25 @@
 
 import com.nulabinc.zxcvbn.matchers.Match;
 
+import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
 public class Zxcvbn {
 
+    private final Context context;
+
     public Zxcvbn() {
+        try {
+            this.context = new ZxcvbnBuilder().buildContext();
+        } catch (IOException e) {
+            throw new IllegalStateException(e);
```

Comment:
It is already Wrapped with IllegalStateException in buildContext as follows.
Is it necessary to CATCH here as well?

```
Context buildContext() throws IOException {
    try {
        // ...
        return new Context(dictionaryMap, keyboardMap);
    } catch (IOException e) {
        throw new IllegalStateException(e);
    }
}
```